src/chrome/Button.tsx

```diff
@@ -45,7 +50,8 @@ const Button: React.FC<ButtonProps> = ({
         'text-white bg-red-600 hover:bg-red-700 focus:ring-red-500': (type === 'danger'),
       }
     )}
-    onClick={onClick}
+    onClick={disabled ? () => {} : onClick}
```

Comment:
do we need to pass a noop function here? seems like `onClick={!disabled && onClick}` would make more sense